tcf_website/templates/course/course.html

```diff
@@ -17,6 +17,7 @@
             <h1 class="mr-md-3">{{ course.code }}</h1>
             <h2>{{ course.title }}</h2>
         </div>
+        <button type="button" ><a href={{course.eval_link}}>Student Evaluations (Requires NetBadge login)</a></button>
```

Comment:
Maybe make this a boostrap button for now? Unless you want to wait until you hear from UX people

Reply:
Yeah I don't have a lot of experience w/ Bootstrap so I figured I'd wait to sort that out until I know exactly what the target is